kafka-ui-react-app/src/components/Brokers/BrokersContainer.ts

```diff
@@ -1,20 +1,24 @@
 import { connect } from 'react-redux';
-import {
-  fetchBrokers,
-  fetchBrokerMetrics,
-} from 'redux/actions';
-import Brokers from './Brokers';
+import { fetchBrokers, fetchBrokerMetrics } from 'redux/actions';
 import * as brokerSelectors from 'redux/reducers/brokers/selectors';
 import { RootState, ClusterName } from 'redux/interfaces';
 import { RouteComponentProps } from 'react-router-dom';
+import Brokers from './Brokers';
 
 interface RouteProps {
   clusterName: ClusterName;
 }
 
-interface OwnProps extends RouteComponentProps<RouteProps> { }
+type OwnProps = RouteComponentProps<RouteProps>;
 
-const mapStateToProps = (state: RootState, { match: { params: { clusterName } }}: OwnProps) => ({
+const mapStateToProps = (
+  state: RootState,
+  {
+    match: {
+      params: { clusterName },
+    },
+  }: OwnProps
+) => ({
```

Comment:
why is it better? Somewhere you collapse objects, and somewhere you don't. What logic should we follow? only the length of the line?